src/BloomExe/Program.cs

```diff
@@ -250,9 +250,17 @@ static void Main(string[] args1)
 #if DEBUG
 						StartDebugServer();
 #endif
-						L10NSharp.LocalizationManager.SetUILanguage(Settings.Default.UserInterfaceLanguage, false);
+						LocalizationManager.SetUILanguage(Settings.Default.UserInterfaceLanguage, false);
+
+						// BL-1258: sometimes the newly installed fonts are not available until after Bloom restarts
+						if (FontInstaller.InstallFont("AndikaNewBasic"))
+						{
+							var restartMsg = LocalizationManager.GetDynamicString("Bloom", "Startup.FontInstalled",
+								"New fonts have been installed. Bloom will shut down now, and when you restart it the fonts will be available.");
```

Comment:
Let's avoid passive voice here, which requires more advanced English skills. So let's say "Bloom has installed a new font."  (singular because though technically we installed 1 typeface and 3 or so fonts, users think of font==typeface).

Q: is there a reason we can't actually restart for them?